NEWS.md

```diff
@@ -0,0 +1,4 @@
+# rhino 1.1.1.9999
```

Comment:
Thanks for adding this file!

Let's follow the structure used by [tidyverse](https://github.com/tidyverse/glue/blob/main/NEWS.md) and [r-lib](https://github.com/r-lib/devtools/blob/main/NEWS.md). When making a new release, we can then just replace this header with the appropriate version number.
```suggestion
# rhino (development version)
```